src/test/java/tests/SmokeTest.java

```diff
@@ -0,0 +1,17 @@
+package tests;
+
+import baseEntities.BaseTest;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import pages.MainPage;
+
+public class SmokeTest extends BaseTest {
+
+    @Test
```

Comment:
add description with details about the test inside @Test annotation

Reply:
done